### PR TITLE
Longw/window vul

### PR DESCRIPTION
**Pull request by wanlonghenry**
Longw/window vulnerability fix for https://nvd.nist.gov/vuln/detail/CVE-2013-3900

### CODE REVIEW

**Comment by wanlonghenry** [comment]
image

**Review by jatakiajanvi12 — COMMENTED** [review]
Hey @wanlonghenry not able to reach the vulnerability link in description. Can you update the link?

File: kubernetes/windows/Dockerfile [hunk 1]
```diff
@@ -5,6 +5,9 @@ LABEL maintainer="OMSContainers@microsoft.com"
 LABEL vendor=Microsoft\ Corp \
     com.microsoft.product="Azure Monitor for containers"
 
+RUN reg add "HKLM\Software\Microsoft\Cryptography\Wintrust\Config" /v "EnableCertPaddingCheck" /t REG_DWORD /d "1" /f
+RUN reg add "HKLM\Software\Wow6432Node\Microsoft\Cryptography\Wintrust\Config" /v "EnableCertPaddingCheck" /t REG_DWORD /d "1" /f
```
Comment (ganga1980) [inline kubernetes/windows/Dockerfile:9]: change, lgtm. valide this change on both ws2019 and ws2022, and also verify this change addresses the reported vuln.

**Comment by wanlonghenry** [comment]
> Hey @wanlonghenry not able to reach the vulnerability link in description. Can you update the link?

updated

**Comment by wanlonghenry** [comment]
image

File: kubernetes/windows/Dockerfile (hunk 1 above)
Comment (wanlonghenry) [reply kubernetes/windows/Dockerfile:9]: verified

**Review by ganga1980 — APPROVED** [review]

**Pull request merged**